app/models/course/condition.rb

```diff
@@ -3,4 +3,6 @@ class Course::Condition < ActiveRecord::Base
 
   belongs_to :course, inverse_of: false
   belongs_to :conditional, polymorphic: true
+
+  delegate :satisfied_by?, to: :actable
```

Comment:
Sorry, I don't really understand your question. `Course::Condition` is the base class for all conditions so it delegates down to the specific class which is `actable`.